### PR TITLE
adjusted hmtl route to account page, added 4 more tamagotchi avatars

### PR DESCRIPTION
**Pull request by dirk-kiesewetter**

### CODE REVIEW

File: routes/htmlRoutes.js [hunk 1]
```diff
@@ -3,59 +3,58 @@ var db = require("../models");
...
       res.render("account", {
         email: req.user.email,
         id: req.user.id,
         createdAt: req.user.createdAt.substring(0, 10)
-      })
+      });
     } else if (!req.user) {
-      res.render("index")
-    };
+      res.render("account");
```
Comment (plsenh) [inline routes/htmlRoutes.js]: should keep this part to go to index if user is not authenticated (not signed in)

File: routes/htmlRoutes.js [hunk 2]
```diff
@@ -3,59 +3,58 @@ var db = require("../models");
...
-  app.get("/account", function (req, res) {
+  app.get("/account", function(req, res) {
     if (req.user) {
       res.render("account", {
         email: req.user.email,
         id: req.user.id,
         createdAt: req.user.createdAt.substring(0, 10)
-      })
+      });
     } else if (!req.user) {
```
Comment (plsenh) [inline routes/htmlRoutes.js]: I forgot to update this part to just say `else` instead of `else if (!req.user)`

File: routes/htmlRoutes.js (hunk 1 above)
Comment (dirk-kiesewetter) [reply routes/htmlRoutes.js]: Done.

File: routes/htmlRoutes.js (hunk 2 above)
Comment (dirk-kiesewetter) [reply routes/htmlRoutes.js]: And done.

**Pull request merged**